packages/client-sdk-nodejs/src/config/configurations.ts

```diff
@@ -154,6 +157,47 @@ class InRegionLowLatency extends CacheConfiguration {
   }
 }
 
+class InRegionLambda extends CacheConfiguration {
+  /**
+   * Provides the latest recommended configuration for a in-region lambda environment.  NOTE: this configuration may
+   * change in future releases to take advantage of improvements we identify for default configurations.
+   * @param {MomentoLoggerFactory} [loggerFactory=defaultLoggerFactory]
+   * @returns {CacheConfiguration}
+   */
+  static latest(
+    loggerFactory: MomentoLoggerFactory = defaultLoggerFactory
+  ): CacheConfiguration {
+    return InRegionLambda.v1(loggerFactory);
+  }
+
+  /**
+   * Provides v1 recommended configuration for an in-region lambda environment.  This configuration is guaranteed not
+   * to change in future releases of the Momento node.js SDK.
+   * @param {MomentoLoggerFactory} [loggerFactory=defaultLoggerFactory]
+   * @returns {CacheConfiguration}
+   */
+  static v1(
+    loggerFactory: MomentoLoggerFactory = defaultLoggerFactory
+  ): CacheConfiguration {
+    const deadlineMillis = 1100;
```

Comment:
definitely not LowLatency. If we were going to change it from `InRegion.Default` I think it would be to increase it, not decrease it, but my gut is to go with the same value we use for `InRegion.Default`.

@eaddingtonwhite lmk if you have any super strong opinions here.